src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -142,6 +142,12 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         return new Promise(async (resolve, reject) => {
 
             switch (paymentMethod.method) {
+                case StripeElementType.Alipay:
+                    ({error, paymentIntent} = await this._getStripeJs().confirmAlipayPayment(
+                        clientSecret, { return_url: returnUrl }, { handleActions: false }));
```

Comment:
I can see `handleActions` here is always false, but the documentation further down explains you can decide to handle actions yourself?

Reply:
That's correct, @capsula4. By default, StripeJs library handles redirect by calling ConfirmAlipayPayment and this doesn't suits our redirect flow. This is why we must handle that piece ourselves.